features/addressing/managing_countries/adding_country_with_provinces.feature

```diff
@@ -0,0 +1,30 @@
+@addressing
+Feature: Adding countries with provinces
+    In order to sell my goods to different countries and provinces
+    As an Administrator
+    I want to add a new country with its provinces to the store
+
+    Background:
+        Given I am logged in as an administrator
+
+    @ui @javascript
+    Scenario: Adding a country with a province
+        Given I want to add a new country with a province
+        When I select "United Kingdom"
+        And I add the "Scotland" province with "GB-SCT" code
+        And I add it
+        Then I should be notified about successful creation
+        And the country "United Kingdom" should appear in the store
+        And this country should have the "Scotland" province
+
+    @ui @javascript
+    Scenario: Adding a country with two provinces
+        Given I want to add a new country with a province
+        When I select "United Kingdom"
+        And I add the "Scotland" province with "GB-SCT" code
+        And I add the "Nothern Ireland" province with "GB-NIR" code and "N.Ireland" abbreviation
+        And I add this country
+        Then I should be notified about successful creation
+        And the country "United Kingdom" should appear in the store
+        And this country should have the "Scotland" province
```

Comment:
Nope it won't resolve problem, we should change step definition with something like this `the country "United Kingdom" should appear in the store with "Scotland" province`